src/getHumanAge.test.js

```diff
@@ -8,6 +8,26 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge(11, 12)).toEqual([0, 0]);
+  });
+
+  it(`should return an array`, () => {
+    expect(getHumanAge(16, 17)).toEqual([1, 1]);
+  });
+
+  it(`should return an array`, () => {
+    expect(getHumanAge(25, 28)).toEqual([2, 2]);
+  });
 
+  it(`should return an array`, () => {
+    expect(getHumanAge(30, 29)).toEqual([3, 3]);
+  });
+
+  it(`should return an array`, () => {
+    expect(getHumanAge(40, 54)).toEqual([6, 8]);
+  });
+
+  it(`should return an array`, () => {
```

Comment:
This test checks the function properly, although I'm not sure if it has relevant input, considering the fact that cats and humans don't live that long.